ui/tests/acceptance/secrets/backend/gcp/gcp-configuration-test.js

```diff
@@ -20,91 +21,235 @@ import {
   expectedConfigKeys,
   expectedValueOfConfigKeys,
   configUrl,
+  fillInGcpConfig,
 } from 'vault/tests/helpers/secret-engine/secret-engine-helpers';
 
 module('Acceptance | GCP | configuration', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(function () {
+    const flash = this.owner.lookup('service:flash-messages');
     this.store = this.owner.lookup('service:store');
+    this.flashSuccessSpy = spy(flash, 'success');
+    this.flashDangerSpy = spy(flash, 'danger');
+    this.flashInfoSpy = spy(flash, 'info');
     this.version = this.owner.lookup('service:version');
     this.uid = uuidv4();
     this.type = 'gcp';
+    this.path = `GCP-${this.uid}`;
     return authPage.login();
   });
-  module('isEnterprise', function (hooks) {
+
+  test('it should prompt configuration after mounting the GCP engine', async function (assert) {
+    await visit('/vault/settings/mount-secret-backend');
+    await mountBackend(this.type, this.path);
+
+    assert.strictEqual(
+      currentURL(),
+      `/vault/secrets/${this.path}/configuration`,
+      'navigated to configuration view'
+    );
+    assert.dom(GENERAL.emptyStateTitle).hasText('Google Cloud not configured');
+    assert.dom(GENERAL.emptyStateActions).hasText('Configure Google Cloud');
+    // cleanup
+    await runCmd(`delete sys/mounts/${this.path}`);
+  });
+
+  test('it should transition to configure page on click "Configure" from toolbar', async function (assert) {
+    await enablePage.enable(this.type, this.path);
+    await click(SES.configure);
+    assert.strictEqual(
+      currentURL(),
+      `/vault/secrets/${this.path}/configuration/edit`,
+      'navigated to configuration edit view'
+    );
+    // cleanup
+    await runCmd(`delete sys/mounts/${this.path}`);
+  });
+
+  module('isCommunity', function (hooks) {
```

Comment:
nit: I think we tend to make module names human readable and not written like variables. So I'd expect this to just be "Community"